catalog-info.yaml

```diff
@@ -0,0 +1,79 @@
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-elastic-package
+  description: ':go: Build and test the elastic-package project'
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/elastic-package
+
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: elastic-package
+      description: ':go: Build and test the elastic-package project'
+    spec:
+      branch_configuration: "main v0.* v1.*"
+      pipeline_file: ".buildkite/pipeline.yml"
+      provider_settings:
+        build_pull_request_forks: false
```

Comment:
should we have this set to true?

Reply:
If it is set to true, pull requests created by external contributors would run the pipelines without any check.

Builds from pull requests are created through the [buildkite-pr-bot](https://github.com/elastic/buildkite-pr-bot): https://github.com/elastic/elastic-package/blob/main/.buildkite/pull-requests.json

This bot needs to be set in each repository 
https://docs.elastic.dev/ci/configuring-pull-requests-for-buildkite